source/docs/software/getting-started/new-for-2020.rst

```diff
@@ -1,13 +1,81 @@
 New for 2020
 ============
 
-WPILib developers have done a number of changes to the control system for the 2020 season. This article will describe and provide a brief overview of the new changes and features. 
+WPILib developers have done a number of changes to the control system for the 2020 season. This article will describe and provide a brief overview of the new changes and features.
+
+frc-docs/ScreenSteps
+--------------------
+
+- ScreenSteps has been replaced with `frc-docs <https://docs.wpilib.org>`__. This change has largely been inspired by growing demands by the WPILib team. We hope that this move toward a community-led model will increase the quality of tutorials and documentation moving forward.
+- Code examples are testable again WPILib releases, ensuring they are working and accurate.
+- Link Checking. Every link in frc-docs gets the link tested to ensure validity.
 
 WPILib
 ------
 
-- LinearDigitalFilter has been renamed to LinearFilter, and now has a ``Calculate()`` method which returns the filtered value
-- PIDController has been rewritten
-  - Takes lambdas for the measurements and outputs
-  - No longer runs asynchronously. Async has been a footgun for a large majority of teams, and a new implementation will be implemented in the future.
-- Kinematics classes have been added for Swerve, Mecanum, and DifferentialDrive. This allows for more complete closed loop driving of these drive types.
\ No newline at end of file
+The core WPILib library has received many changes. These include rewrites to major part of the core library, as well as a suite of kinematics to improve closed loop driving. The full change log can be read below.
+
+.. todo:: Add links to the specific usage guides for LinearDigitalFilter, and PIDController.
+
+- CommandBased has been rewritten from the ground up. The explanation on why can be found `here <https://github.com/Oblarg/docs/blob/master/CommandRewriteDesignDoc.md>`__.
```

Comment:
@Oblarg Can you open a PR to add this document to https://github.com/wpilibsuite/design-docs

Then we can update this link to point at the design docs repo.

Reply:
Done.